Feature/explainability comparison

seemab24 opened the pull request:
This Pull Request adds the following features:
- Scaled the training and test data using `StandardScaler`.
- Added SHAP and LIME comparison functionalities for model explainability.
- Updated the model training process to use the scaled data.

Please review the code and provide feedback. Let me know if any adjustments are needed!

ombhojane reviewed (changes requested):
hey please have tis project under /examples dir. up there create a meaningful project name as /explainability_comparison. then have all your codes and notebooks in that dir. Readme file would be a plus explaining about brief about project.

Thanks for contributing, looking to merge this PR

seemab24 commented:
Hi @ombhojane , I have updated the README and added the LIME and SHAP explanations, along with new model comparison steps. Please review the changes. Thank you!
